packages/@angular/cli/commands/e2e.ts

```diff
@@ -14,6 +14,7 @@ export interface E2eTaskOptions extends ServeTaskOptions {
   webdriverUpdate: boolean;
   specs: string[];
   elementExplorer: boolean;
+  useDist: boolean;
```

Comment:
Yeah, I questioned calling it this, but I wasn't sure what the right naming would be. I want something that says "use existing build" but I'm not sure the best way to express that. Should it just be `--use-output-path` (feels a bit verbose to me). Any ideas?